torch/_export/__init__.py

```diff
@@ -120,6 +127,18 @@ class ExportDynamoConfig:
 DECOMP_TABLE = core_aten_decompositions()
 
 
+def capture_pre_autograd_graph(
```

Comment:
can you add some docs about plans for this API

Reply:
just added